lazer/sdk/rust/client/src/client.rs

```diff
@@ -124,16 +247,43 @@ impl PythLazerClient {
     }
 }
 
+/// A builder for creating [`PythLazerClient`] instances with customizable configuration.
+///
+/// The builder provides a convenient way to configure a Pyth Lazer client with sensible
+/// defaults while allowing customization of all parameters. It follows the builder pattern
+/// for a fluent API.
+///
+/// ## Default Configuration
+///
+/// - **Endpoints**: Uses Pyth Lazer's default production endpoints
+/// - **Connections**: 4 concurrent WebSocket connections
+/// - **Timeout**: 5 seconds for WebSocket operations
+/// - **Backoff**: Exponential backoff with default settings
+/// - **Channel Capacity**: Uses the default 1000
+///
 pub struct PythLazerClientBuilder {
     endpoints: Vec<Url>,
     access_token: String,
     num_connections: usize,
-    backoff: ExponentialBackoff,
+    backoff: PythLazerExponentialBackoff,
     timeout: Duration,
     channel_capacity: usize,
 }
 
 impl PythLazerClientBuilder {
+    /// Creates a new builder with default configuration.
+    ///
+    /// This initializes the builder with sensible defaults for production use:
+    /// - Default Pyth Lazer endpoints
+    /// - 4 WebSocket connections
+    /// - 5-second timeout
+    /// - Default exponential backoff settings
+    /// - Default channel capacity
```

Comment:
Doesn't make sense to list the last two without providing actual values.

Reply:
removed them.